scylla.go

```diff
@@ -80,12 +80,14 @@ func isScyllaConn(conn *Conn) bool {
 
 // scyllaConnPicker is a specialised ConnPicker that selects connections based
 // on token trying to get connection to a shard containing the given token.
+// A list of excess connections is maintained to allow for lazy garbage collection
 type scyllaConnPicker struct {
```

Comment:
Would be nice to explain why

Reply:
Yes it's a bit sparse, I will amend the doc and commit message.

Reply:
Updated both in the code and the commit message.